### ISSUE TITLE
Plugin Toggler does not work on some Touch-Devices (e.g. iPhone 6, Samsung S3)

### ISSUE THREAD

**Issue by michaelscheel** [zurb/foundation-sites#8250]
When using the plugin toggler on iPhone 6 or older Android devices it does not work (adds no class to the target element).

If I modify foundation.util.triggers.js from:

```
  // Elements with [data-toggle] will toggle a plugin that supports it when clicked.
  $(document).on('click.zf.trigger', '[data-toggle]', function() {
    var id = $(this).data('toggle');
    $('#' + id).triggerHandler('toggle.zf.trigger', [$(this)]);
  });
```

...to:

```
  // Elements with [data-toggle] will toggle a plugin that supports it when clicked.
  $(document).on('click.zf.trigger touchstart.zf.trigger', '[data-toggle]', function() {
    var id = $(this).data('toggle');
    $('#' + id).triggerHandler('toggle.zf.trigger', [$(this)]);
  });
```

...it works.

**Comment by michaelvolz** [comment]
@michaelscheel Nice fix, thank you. 
I wonder if there is an easy way to only accept the touch toggle when it is just a short tap and not a long slide while scrolling the page. Any ideas?

**Comment by AleksandarSto** [comment]
Hello, 

I'm currently facing the same issue with my website. Toggler seems to be working fine on desktop and desktop-simulated mobile devices.

However, when I load the page on my Android phone, using Chrome, the toggler button no longer works and the field which is supposed to be hidden by default is displayed. 

I want to try your fix, but the website was developed in way where foundation.util.triggers.js is imported via another file called - foundation-explicit-pieces.js - for which I couldn't find any documentation. 

I did not develop the website and I'm not really a developer either, so any help will be much appreciated. Trying to wrap my head around this, but it's really beyond my abilities to solve.

**Comment by DanielRuf** [comment]
Can you provide a working codepen?

**Comment by AleksandarSto** [comment]
Sorry, I can't get it to work on CodePen at all - https://codepen.io/anon/pen/JejNMp?editors=1100

Here is a sample of my HTML structure with the elements in question. It's funny that it works halfway on my website and not at all on codepen.

**Comment by DanielRuf** [comment]
Works for me at https://codepen.io/DanielRuf/pen/RqwZXM

**Comment by DanielRuf** [comment]
You probably just forgot `$(document).foundation();` in your codepen.

**Comment by AleksandarSto** [comment]
Yeah, I got it working in CodePen too.

But the real thing which is identical in structure refuses to work on Chrome for Android. The browser actually refuses to hide the field in the first place. 

I found another user who has had the same problem as me, but no mention of the solution.  https://stackoverflow.com/questions/37308862/display-none-on-android

Likely stems out of me trying to do things I'm not fit to be doing.
